README.md

```diff
@@ -173,6 +173,49 @@ When using Coverage.jl locally, over time a lot of `.cov` files can accumulate.
        - C:\projects\julia\bin\julia -e "using Pkg; Pkg.add(\"Coverage\"); using Coverage; Coveralls.submit(process_folder())"
        ```
 
+## A note for advanced users
+
+Coverage tracking in Julia is not yet quite perfect. One problem is that (at
+least in certain scenarios), the coverage data emitted by Julia does not mark
+functions which are never called (and thus are not covered) as code. Thus,
+they end up being treated like comments, and are *not* counted as uncovered
+code, even though they clearly are. This can arbitrarily inflate coverage
+scores, and in the extreme case could even result in a project showing 100%
+coverage even though it contains not a single test.
+
+To overcome this, Coverage.jl applies a workaround which ensures that all
+lines of code in all functions of your project are properly marked as "this is
+code". This resolves the problem of over reporting coverage. 
+
+Unfortunately, this workaround itself can have negative consequences, and lead
+to under reporting coverage, for the following reason: when Julia compiles
+code with inlining and optimizations, it can happens that some lines of Julia
+code get optimized away resp. folded into other code; in that case, Julia's
```

Comment:
I'm not sure what "optimized away resp. folded into other code" means.

Reply:
That was indeed written in an unclear fashion. How about this:  "... it can happen that some lines of Julia code do not correspond to any generated machine code; in that case ..."

BTW, for an example, see the lines following https://codecov.io/gh/JuliaCI/Coverage.jl/src/master/src/codecovio.jl#L184

Reply:
I was mostly confused by "resp." but I like your proposed reword.

Reply:
Updated the PR